test/visualTest.ts

```diff
@@ -432,6 +433,112 @@ describe("AsterPlot", () => {
             });
         });
 
+        function timeout(ms: number) {
+            return new Promise(resolve => setTimeout(resolve, ms));
+        }
+
+        const DefaultWaitForRender: number = 500;
+
+        describe("Keyboard Navigation check", () => {
+            it("links should have attributes tabindex=0, role=option and aria-selected=false", (done) => {
+                visualBuilder.updateRenderTimeout(dataView, () => {
+                    visualBuilder.updateFlushAllD3Transitions(dataView);
+                    let nodes = [...visualBuilder.slices];
+                    nodes.forEach((el: Element) => {
+                        expect(el.getAttribute("role")).toBe("option");
+                        expect(el.getAttribute("tabindex")).toBe("0");
```

Comment:
Once again, it might me a better idea to increment tabindex, see
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex#try_it